### PR TITLE
Added NOTICE file, removed dependency that wasn't required.

### PR DESCRIPTION
**Pull request by lyndonbauto**
*Issue #, if available:*
https://bitquill.atlassian.net/browse/AN-1046

*Description of changes:*
Added NOTICE file, removed dependency that wasn't required and was unlicensed.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: gremlin-go/NOTICE [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+====================================================================================================
+Indirectly Used Licenses:
+====================================================================================================
+ISC (functionally equivalent to BSD-2):
+    github.com/davecgh/go-spew v1.1.0 => https://github.com/davecgh/go-spew/blob/v1.1.0/LICENSE
+MIT:
+	github.com/stretchr/objx v0.1.0 => https://github.com/stretchr/objx/blob/v0.1.0/LICENSE
+MIT and Apache 2.0 (dual license):
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c => https://github.com/go-yaml/yaml/blob/v3/LICENSE
```
Comment (lyndonbauto) [inline gremlin-go/NOTICE]:
```suggestion
MIT:
    github.com/stretchr/objx v0.1.0 => https://github.com/stretchr/objx/blob/v0.1.0/LICENSE
MIT and Apache 2.0 (dual license):
    gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c => https://github.com/go-yaml/yaml/blob/v3/LICENSE
```

File: gremlin-go/driver/connection_test.go [hunk 2]
```diff
@@ -133,20 +134,51 @@ func readCount(t *testing.T, g *GraphTraversalSource, label string, expected int
...
 	results, err := g.V().Fold().
 		Project(testLabel, personLabel).
-			By(T__.Unfold().HasLabel(testLabel).Count()).
-			By(T__.Unfold().HasLabel(personLabel).Count()).
+		By(T__.Unfold().HasLabel(testLabel).Count()).
+		By(T__.Unfold().HasLabel(personLabel).Count()).
 		ToList()
 	assert.Nil(t, err)
-	assert.Equal(t,1, len(results))
+	assert.Equal(t, 1, len(results))
```
Comment (lyndonbauto) [inline gremlin-go/driver/connection_test.go]: Auto format updated this.

File: gremlin-go/driver/connection_test.go [hunk 3]
```diff
@@ -133,20 +134,51 @@ func readCount(t *testing.T, g *GraphTraversalSource, label string, expected int
...
+		return value
+	}
+	return defaultValue
+}
+
+func getEnvOrDefaultBool(key string, defaultValue bool) bool {
+	value := getEnvOrDefaultString(key, "")
+	if len(value) != 0 {
+		boolValue, err := strconv.ParseBool(value)
+		if err != nil {
```
Comment (vkagamlyk) [inline gremlin-go/driver/connection_test.go]: Why returned boolValue when error has occurred?

Comment (lyndonbauto) [reply gremlin-go/driver/connection_test.go]: Oops, this was supposed to read err == nil. Error is repeated below, i will fix this..

Comment (lyndonbauto) [inline gremlin-go/driver/connection_test.go]:
```suggestion
		if err == nil {
```

File: gremlin-go/driver/connection_test.go [hunk 4]
```diff
@@ -133,20 +134,51 @@ func readCount(t *testing.T, g *GraphTraversalSource, label string, expected int
...
+		}
+	}
+	return defaultValue
+}
+
+func getEnvOrDefaultInt(key string, defaultValue int) int {
+	value := getEnvOrDefaultString(key, "")
+	if len(value) != 0 {
+		intValue, err := strconv.Atoi(value)
+		if err != nil {
```
Comment (lyndonbauto) [inline gremlin-go/driver/connection_test.go]:
```suggestion
		if err == nil {
```

**Review by L0Lmaker — APPROVED** [review]

**Comment by L0Lmaker** [comment]
There's a merge conflict but otherwise LGTM

**Review by xiazcy — APPROVED** [review]
LGTM

**Review by simonz-bq — APPROVED** [review]

**Pull request merged**